packages/proxygen/src/uuidHelper.ts

```diff
@@ -0,0 +1,7 @@
+import {v4 as uuidv4} from "uuid"
+
+// this is a separate function to allow it to be mocked
```

Comment:
why not mock the uuid package directly?

Reply:
uuid hates being mocked with jest and latest typescript. 
Mocking it directly leads to it either not being mocked, or it failing typescript compilation when using jest